storage/zynq7000-flash/flashdrv.h

```diff
@@ -30,9 +32,9 @@ extern int flashdrv_done(storage_t *strg);
 
 
 /* Initialize only a one physicall flash memory via qspi interface, returns:
-    - >0 - number of remaining logic devices for initialization (in case where there are multiple regions)
-    - <0 - on error
-    - 0  - all logic devices are initialized
+	- >0 - number of remaining logic devices for initialization (in case where there are multiple regions)
+	- <0 - on error
+	- 0  - all logic devices are initialized
```

Comment:
```suggestion
* - >0 - number of remaining logic devices for initialization (in case where there are multiple regions)
* - <0 - on error
* - 0  - all logic devices are initialized
```

Reply:
Just clang-format acting up again and I've missed it.